scripts/Tools/save_provenance

```diff
@@ -71,6 +71,7 @@ def _main_func(description):
                 expected_timing_file = os.path.join(caseroot, "timing", "%s_timing.%s.%s.gz" % (model, caseid, lid))
                 if (not os.path.exists(expected_timing_file)):
                     get_timing(case, lid)
+                save_prerun_provenance(case, lid=lid)
```

Comment:
When the user decides post-run that they want to save timings, we'll need to save the pre-run stuff in addition to post-run since none of it would have been saved originally.